spec/functional/resource/group_spec.rb

```diff
@@ -24,17 +24,19 @@
   include Chef::Mixin::ShellOut
 
   def group_should_exist(group)
-    case ohai[:os]
-    when "linux"
-      expect { Etc.getgrnam(group) }.not_to raise_error
-      expect(group).to eq(Etc.getgrnam(group).name)
+    case ohai[:platform]
+    when "freebsd"
+      expect(shell_out("pw groupshow -n #{group}").exitstatus).to eq(0)
     when "windows"
       expect { Chef::Util::Windows::NetGroup.new(group).local_get_members }.not_to raise_error
+    else
+      expect { Etc.getgrnam(group) }.not_to raise_error
+      expect(group).to eq(Etc.getgrnam(group).name)
     end
   end
 
   def user_exist_in_group?(user)
-    case ohai[:platform_family]
+    case ohai[:platform]
```

Comment:
Small request, but maybe keep the platforms in alphabetical order so that if you're scanning which OSes are handled you can see at a glance?